eurec4a_environment/nomenclature.py

```diff
@@ -0,0 +1,118 @@
+"""
+To ensure that all functions in eurec4a-environment can access the variables
+required the module uses a common nomenclature for variable names throughout.
+To add a new variable to the nomenclature simply a) add a new constant below
+setting the assumed field name and b) add (if available) the CF-conventions
+standard name mapping in `CF_STANDARD_NAMES` if the variable has a "standard
+name" (See http://cfconventions.org/standard-names.html for the list of
+"standard names")
+"""
+import inspect
+import xarray as xr
+
+
+TEMPERATURE = "T"
```

Comment:
```suggestion
TEMPERATURE = "ta"
```
This is just to be consistent with the latest variable names in JOANNE - most variable names are finalised, but some are still being discussed.. :/

Reply:
No problem :)